### ISSUE TITLE
Focus lost on search

### ISSUE THREAD

**Issue by eco747** [Komodo/KomodoEdit#805]
When i do a search on a document (Ctrl+F) and i dismiss the search bar (ESC), the focus is lost.
I must use my mouse to go on the document.

Windows 10.

by the way, is there any chance to be able to change case matching, whole word or other options with keyboard in the search bar ?

**Comment by Naatan** [comment]
Not reproducible on my machine. Try opening a new tab (Ctrl+T) then hit the Troubleshooting button and then "Restart in Safe-Mode", see if you can reproduce the issue in Safe-Mode.

There is currently no way to toggle search options with the keyboard, which is sad since keyboard support is a big focus for us. Please consider filing an enhancement request.

**Comment by eco747** [comment]
Hi,
I can reproduce it even in safe mode. But i have to tell you that i'm using
the Edit/Find command not the 'Incremental find' (Ctrl+F)

1. new tab with text
[image: Images intégrées 1]

2. Edit/Find
[image: Images intégrées 2]

3. search something & press enter
[image: Images intégrées 3]

4. text is found so press esc to close the small search bar

-> no focus : i need my mouse to continue editing.
-> beeeuuuaaa

regards
Etienne


2015-11-05 17:12 GMT+01:00 Nathan Rijksen <notifications@github.com>:

**Comment by Naatan** [comment]
Again, please respond using github :) Email responses get garbled.

**Comment by Naatan** [comment]
Reproduced using Edit > Find. Thanks! :)

**Comment by eco747** [comment]
By the way, after the focus is lost, if you switch to another application, come back to Komodo, then press Ctrl+F, you get an exception
I think this will be ok after the correction.

<img width="445" alt="im5" src="https://cloud.githubusercontent.com/assets/2598404/10977655/9f95ef50-83f2-11e5-82aa-903238a20360.png">

**Comment by testworksau** [comment]
Will this also fix the issue with replacing text?

If I highlight some code, then press CTRL-H, then press tab and make a change to the Replace field:

 - Sometimes Replace All does nothing. I can click it multiple times but nothing changes
 - After clicking Replace All, sometimes clicking on the code in the editor doesn't do anything - i.e. the editor does not gain focus and the find / replace window remains open.

I think the second problem may be the python linters / syntax checkers blocking access - the reason I think this is the case is because the squiggly lines disappear after the Replace All operation, and it's not until they reappear that I get control back.

**Comment by Naatan** [comment]
That sounds like a different bug, please open a new ticket for it.

**Comment by testworksau** [comment]
Done, thanks Naatan.

**Comment by agroszer** [comment]
this bugs me too
by default shift+ctrl+e to return the focus to the editor
